packages/camera/camera/ios/Classes/CameraPlugin.m

```diff
@@ -347,10 +382,7 @@ - (void)captureOutput:(AVCaptureOutput *)output
     }
   }
   if (!CMSampleBufferDataIsReady(sampleBuffer)) {
-    _eventSink(@{
-      @"event" : @"error",
-      @"errorDescription" : @"sample buffer is not ready. Skipping sample"
-    });
+    [_methodChannel invokeMethod:@"error" arguments:@"sample buffer is not ready. Skipping sample"];
```

Comment:
nit: I think `@"error"` would be better as a constant like it is in Java so it doesn't need to be copy and pasted everywhere.

Reply:
Agreed, updated the code to use a constant called `errorMethod` and use it where we do a `invokeMethod:@"error"`.